tensorflow/workspace.bzl

```diff
@@ -172,6 +172,17 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+      name = "mkl",
+      urls = [
+          "https://github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
+          "http://mirror.bazel.build/github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
```

Comment:
The downloader is designed to *always* try the first URL in the list, and then begins to fail over in parallel to the second URL after like 2000ms. So the fastest URL should always be the first URL.